composer/algorithms/sam/__init__.py

```diff
@@ -1,12 +1,13 @@
 # Copyright 2021 MosaicML. All Rights Reserved.
 
+"""SAM (`Foret et al, 2020 <https://arxiv.org/abs/2010.01412>`_) wraps an existing optimizer
+with a :class:`SAMOptimizer` which makes the optimizer minimize both loss value and sharpness.
```

Comment:
Link to class not resolving. Try: 
```suggestion
with a :class:`~composer.algorithms.sam.sam.SAMOptimizer` which makes the optimizer minimize both loss value and sharpness.
```